editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -350,6 +394,15 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
         return {};
     }
   }
+
+  function getTitleAndTypeFromFlow(nodeId: string) {
```

Comment:
I thought it made sense to look up the flow data in the analytics provider to reduce duplication of this step in the places where the tracking is called. If I'm missing the nuances of interacting with the store state please let me know.

Reply:
I like this approach! Technically possible to do using the store from each component it's called at, but nice to reduce component complexity and shouldn't interfere with applicant-facing response times/performance when done here since analytics requests route through Beacon :+1: